Mage.Tests/src/test/java/org/mage/test/cards/mana/DoublingCubeTest.java

```diff
@@ -0,0 +1,35 @@
+package org.mage.test.cards.mana;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+import org.mage.test.serverside.base.impl.CardTestAPIImpl;
+
+public class DoublingCubeTest extends CardTestPlayerBase {
+
+    String cube = "Doubling Cube";
+    String temple = "Eldrazi Temple";
```

Comment:
minor: add a block comment for Eldrazi Temple since it's important to remember what this card is doing for the test
```
/*
Eldrazi Temple
Land
{T}: Add {C} to your mana pool.
{T}: Add {C}{C} to your mana pool. Spend this mana only to cast colorless Eldrazi spells or activate abilities of colorless Eldrazi.
*/